src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,25 @@
 import React from 'react';
 import './MoviesList.scss';
+import PropTypes from 'prop-types';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ list }) => (
+  <>
+    <div className="movies">
+      {list.map(item => (
+        <MovieCard key={item.imdbId} {...item} />
+      ))}
```

Comment:
Рекомендую трохи розділяти..
```suggestion
      {
        list.map(item => (
          <MovieCard key={item.imdbId} {...item} />
        ))
      }
```

Reply:
no need to do this (but it's a matter of taste, of course)